test/helpers.tsx

```diff
@@ -1,24 +1,118 @@
+import "date-fns";
 import { bigtestGlobals } from "@bigtest/globals";
 import { render as rtlRender } from "@testing-library/react";
-import { ReactElement } from "react";
+import { ComponentProps, ComponentType, ReactElement, useState } from "react";
 import { StylesProvider, jssPreset } from "@material-ui/core/styles";
 import { create } from "jss";
+import { MuiPickersUtilsProvider } from "@material-ui/pickers";
+import DateFnsUtils from "@date-io/date-fns";
 
-export function render(element: ReactElement) {
-  let insertion = bigtestGlobals.document.createComment("mui-jss-insertion");
-  let insertionPoint = bigtestGlobals.document.head.insertBefore(insertion, bigtestGlobals.document.head.firstChild);
-  const jss = create({
-    ...jssPreset(),
-    // Define a custom insertion point that JSS will look for when injecting the styles into the DOM.
-    insertionPoint,
-  });
-
-  return rtlRender(
-    <StylesProvider jss={jss} injectFirst>
-      {element}
-    </StylesProvider>,
-    {
-      container: bigtestGlobals.document.body,
-    }
-  );
+interface StepImplementation {
+  description: string;
+  action: () => void;
+}
+
+export function renderStep(description: ReactElement): StepImplementation;
+export function renderStep(description: string, element: ReactElement): StepImplementation;
+export function renderStep(description: string | ReactElement, element?: ReactElement): StepImplementation {
```

Comment:
Let's just keep this as `render`. 
```suggestion
export function render(description: ReactElement): StepImplementation;
export function render(description: string, element: ReactElement): StepImplementation;
export function render(description: string | ReactElement, element?: ReactElement): StepImplementation {
```

Reply:
It will look strange because it returns a `StepImplementation` and actually doesn't render anything, just create a step. And I don't sure about naming `createRenderStep` because it returns a function that returns a render step. So it more closely to a `createRenderStepFactory`. But I'm ok with your variant :)

Reply:
Ok, I got your point, the `render` looks good when it is used in `test.step(render())` statement